README.md

```diff
@@ -78,9 +78,23 @@ public class LockClientExample {
         client.close();
     }
 }
-
 ```
 
+#### Permissions
+The following permissions are required in order to use the AmazonDynamoDBLockClient:
+- `dynamodb:DeleteItem`
+- `dynamodb:GetItem`
+- `dynamodb:PutItem`
+- `dynamodb:Scan`
+- `dynamodb:UpdateItem`
+
+If you also create a table using `AmazonDynamoDBLockClient.createLockTableInDynamoDB` or want to ensure it exists
+using `AmazonDynamoDBLockClient.assertLockTableExists` or `AmazonDynamoDBLockClient.lockTableExists`, the following
```

Comment:
It would be nice to see these different actions separated out, eg:

```
If you will be using this library to create a table:
- dynamodb:CreateTable`

If you want to verify the table's existence:
- dynamodb:Describe table
```